forge-gui/res/cardsfolder/g/graven_archfiend.txt

```diff
@@ -0,0 +1,9 @@
+Name:Graven Archfiend
+ManaCost:3 B B
+Types:Artifact Creature Gargoyle Demon
+PT:5/4
+K:Flying
+K:Kicker:Sac<1/Creature.nonDemon>
```

Comment:
Try `Sac<1/Creature.nonDemon/non-Demon creature>`

Also check out _Voltage Surge_, couple of things you want to learn from there:
- The Generic is important, because this shouldn't be treated like a real kicker cost for interaction with stuff like _Vine Gecko_
- Needs check on the trigger, try maybe `IsPresent$ Card.Self+pseudokicked | NoResolvingCheck$ True`

Reply:
Those params need to be on the trigger to work correctly (it is "intervening if", see CR 603.4. for details)